mix.exs

```diff
@@ -6,7 +6,7 @@ defmodule Phoenix.Mixfile do
   def project do
     [app: :phoenix,
      version: @version,
-     elixir: "~> 1.3",
+     elixir: "~> 1.4",
```

Comment:
Let's keep the version here `~> 1.3` so we keep people upgrading a more breathing room but people starting a new app start fresh.